packages/react-scripts/config/webpack.config.js

```diff
@@ -264,10 +273,14 @@ module.exports = function(webpackEnv) {
       // Automatically split vendor and commons
       // https://twitter.com/wSokra/status/969633336732905474
       // https://medium.com/webpack/webpack-4-code-splitting-chunk-graph-and-the-splitchunks-optimization-be739a861366
-      /* splitChunks: {
+      splitChunks: isBeameryStaging ? {
```

Comment:
@LGUG2Z has asked that we don't have chunks/code-splitting on staging. This should disable it.

Reply:
👌